src/main/java/util/task/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package util.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline inherits from Task.
+ *
+ * Deadline is a Task that has an added characteristic of a dueDate.
+ */
+public class Deadline extends Task {
+    protected LocalDate dueDate;
+
+    public Deadline(String name, LocalDate details) {
```

Comment:
perhaps a more specific name rather than 'details' such as 'deadline' or 'duedate'

Reply:
Agree with this. `details` does not identify it as a LocalDate object, though as this is just a temporary variable it is not a big issue.